docs/detections/alerts-view-details.asciidoc

```diff
@@ -14,6 +14,12 @@ The alert details flyout contains these informational tabs:
 * *JSON*: The alert data in JSON format.
 * *Osquery Results*: Results from queries attached to rules display on the *Osquery Results* tab. <<add-osquery-response-action,Learn more.>>
 
+The **Share alert** link gives you a way to share alert details. Click the link to copy a URL that you can paste into browsers, cases, messages, and more. We recommend doing this instead of copying the URL from your browser's address bar, which can cause inconsistent results if the Alerts page has filters or relative time ranges. 
```

Comment:
```suggestion
The **Share alert** link gives you a way to share alert details. Click it to copy a URL that you can paste into browsers, cases, messages, and more. We recommend doing this instead of copying the URL from your browser's address bar, which can cause inconsistent results if the Alerts page has filters or relative time ranges. 
```